src/main/java/com/solace/samples/jcsmp/HelloWorld.java

```diff
@@ -69,6 +69,15 @@ public static void main(String... args) throws JCSMPException, IOException {
             properties.setProperty(JCSMPProperties.PASSWORD, args[3]);  // client-password
         }
         properties.setProperty(JCSMPProperties.REAPPLY_SUBSCRIPTIONS, true);  // subscribe Direct subs after reconnect
+
+
+        // If you are using TLS (necessary in the case of Publishing to an OpenShift route port 443 with passthrough) you need these truststore properties.
+        properties.setProperty(JCSMPProperties.SSL_TRUST_STORE, "/home/alee/keystore/clientkeystore");
```

Comment:
Change to `/path/to/keystore` or something like that without alee in it. 
Also, maybe this should be commented out by default or this would cause the app to scale? 

Also in the comment maybe specify that you could also use `javax.net.ssl.trustStore`, `javax.net.ssl.trustStorePassword`, etc. as well?

Reply:
This was a mistake to check in, was just testing some stuff.  Undone.